test/assertions_test.rb

```diff
@@ -0,0 +1,70 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class AssertionsTest < ActionController::TestCase
+      class MyController < ActionController::Base
+        def render_using_serializer
```

Comment:
no need to do all controller tests if the active code is in the serializable resource.  (Also, controller tests should be in the action_controller folder with the others https://github.com/rails-api/active_model_serializers/tree/master/test/action_controller )